shared/src/store/action-reducers/simulation.ts

```diff
@@ -808,6 +907,14 @@ export namespace SimulationActionReducers {
                     );
                 }
 
+                logBehavior(
+                    draftState,
+                    [],
+                    `Das ${reportBehaviorState.type} Verhalten im Bereich ${simulatedRegion.name} wird Informationen vom Typ ${informationType} alle ${interval}ms melden.`,
```

Comment:
You should probably also create an `informationTypeToGermanDictionary` or something similar

Reply:
d

Reply:
Please also use the generated map in the frontend instead of duplicating it :)